docs/migration-guide.md

```diff
@@ -48,101 +46,97 @@ The changes listed below require you to modify your workflow code. The following
 
 ## New features introduced in Prefect 2
 
 The following new components and capabilities are enabled by Prefect 2.
 
 - More flexibility thanks to the elimination of flow pre-registration.
-- More flexibility with respect to flow deployments, including easier promotion of a flow through development, staging, and production environments.
+- More flexibility for flow deployments, including easier promotion of a flow through development, staging, and production environments.
 - Native `async` support.
 - Out-of-the-box `pydantic` validation.
 - [Blocks](../ui/blocks/) allowing you to securely store UI-editable, type-checked configuration to external systems and an easy-to-use Key-Value Store. All those components are configurable in one place and provided as part of the open-source Prefect 2 product. In contrast, the concept of [Secrets](https://docs.prefect.io/orchestration/concepts/secrets.html) in Prefect 1 was much more narrow and only available in Prefect Cloud.  
 - [Notifications](../ui/notifications/) available in the open-source Prefect 2 version, as opposed to Cloud-only [Automations](https://docs.prefect.io/orchestration/ui/automations.html) in Prefect 1.  
-- A first-class `subflows` concept: Prefect 1 only allowed the [flow-of-flows orchestrator pattern](https://discourse.prefect.io/tag/orchestrator-pattern), which is still supported. With Prefect 2 subflows, you are gaining a natural and intuitive way of organizing your flows into modular sub-components. For more details, see [the following list of resources about subflows](https://discourse.prefect.io/tag/subflows).
+- A first-class `subflows` concept: Prefect 1 only allowed the [flow-of-flows orchestrator pattern](https://discourse.prefect.io/tag/orchestrator-pattern). With Prefect 2 subflows, you gain a natural and intuitive way of organizing your flows into modular sub-components. For more details, see [the following list of resources about subflows](https://discourse.prefect.io/tag/subflows).
 
 
 ### Orchestration behind the API
 
 Apart from new features, Prefect 2 simplifies many usage patterns and provides a much more seamless onboarding experience.
 
-Every time you run a flow, it is tracked by the API even if you don’t know it; this brings both local and backend-triggered execution on the exact same page for better debugging and observability. This also allows you to use alternate schedulers (for example, a CRON job) when migrating to Prefect.
-
-Similarly, because the API tracks every flow run behind the scenes, agentless deployments work out-of-the-box &mdash; if you want to manage your own flow execution, you can!
+Every time you run a flow, whether it is tracked by the API server or ad-hoc through a Python script, it is on the same UI page for easier debugging and observability. 
```

Comment:
I think what we mean is how the flow run was started. Perhaps "Every time you run a flow, whether running a Python script on your local machine or a scheduled deployment run via an agent through the API,"

Reply:
Nice!